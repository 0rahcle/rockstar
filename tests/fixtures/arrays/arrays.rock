Let my array at 0 be "foo"
Let my array at 1 be "bar"
Let my array at 2 be "baz"
Shout my array at 0
Shout my array at 1
Shout my array at 2

The line says one two three four five
Split the line into words
The index is 0
The word is out
Until the word is nothing
Put the line at index into the word
Say the word

