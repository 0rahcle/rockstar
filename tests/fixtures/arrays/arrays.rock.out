foo
bar
baz
one
two
three
four
five
